docs/uptime-guide/overview.asciidoc

```diff
@@ -1,11 +1,56 @@
 [role="xpack"]
 [[uptime-overview]]
-== Overview
+== Elastic Uptime overview
 
-The Elastic Uptime solution built on the Elastic Stack.
-It allows you to monitor the availability and response time of software services and applications in real time.
+The Elastic Uptime solution is built on the Elastic Stack.
+Elastic Uptime allows you to monitor the availability and response times of applications and services in real time and to detect problems before they affect users.
+
+Elastic Uptime is based on {heartbeat-ref}/index.html[Heartbeat], an open source uptime monitor.
+Elastic Uptime is transparent, flexible, and developer-friendly.
+It can help you to understand uptime and response time characteristics for your services and applications.
+It can be deployed both inside and outside your organization's network, so you can analyze problems from multiple vantage points.
+
+Elastic Uptime consists of these components: *Heartbeat*, *Elasticsearch* and *Kibana*.
```

Comment:
I wouldn't say that it consists of those parts, but rather that a deployment would require the use of those components.

Reply:
Good point